apps/charterafrica/src/payload/collections/Tools.js

```diff
@@ -0,0 +1,307 @@
+import {
+  TOOL_COLLECTION,
+  ORGANIZATION_COLLECTION,
+  CONTRIBUTORS_COLLECTION,
+} from "../../lib/ecosystem/models";
+import donors from "../fields/donorsField";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const Tools = {
+  slug: TOOL_COLLECTION,
+  admin: {
+    useAsTitle: "externalId",
+    defaultColumns: ["externalId", "name", "location", "description"],
+  },
+  access: {
+    read: () => true,
+  },
+  labels: {
+    singular: {
+      en: "Tool",
+      fr: "Outil",
+      pt: "Ferramenta",
+    },
+    plural: {
+      en: "Tools",
+      fr: "Outils",
+      pt: "Ferramentas",
+    },
+  },
+  fields: [
+    {
+      name: "externalId",
+      type: "text",
+      required: true,
+      label: { en: "External ID", fr: "ID externe", pt: "ID externo" },
+      admin: {
+        readOnly: true,
+      },
+    },
+    slug({ fieldToUse: ["source", "name"] }),
+    {
+      name: "name",
+      type: "text",
+      label: {
+        en: "Tool Name",
+        fr: "Nom de l'outil",
+        pt: "Nome da ferramenta",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "description",
+      type: "textarea",
+      admin: {
+        readOnly: true,
+      },
+      localized: true,
+      label: {
+        en: "Tool Description",
+        fr: "Description de l'outil",
+        pt: "Descrição da ferramenta",
+      },
+    },
+    {
+      name: "link",
+      type: "text",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Link",
+        fr: "Lien de l'outil",
+        pt: "Link da ferramenta",
+      },
+    },
+    {
+      name: "deployedLocation",
```

Comment:
If we're going to track just one deployment location, isn't it better to specify if it's the first deployment / original deployment?